stablehlo/tests/interpret_power.mlir

```diff
@@ -0,0 +1,45 @@
+// RUN: stablehlo-interpreter --interpret -split-input-file %s
+
+// CHECK-LABEL: Evaluated results of function: power_op_test_ui64
+func.func @power_op_test_ui64() {
+  %lhs = stablehlo.constant dense<[0, 0, 1, 1, 5]> : tensor<5xui64>
+  %rhs = stablehlo.constant dense<[0, 1, 0, 2, 5]> : tensor<5xui64>
+  %result = stablehlo.power %lhs, %rhs : tensor<5xui64>
+  check.eq %result, dense<[1, 0, 1, 1, 3125]> : tensor<5xui64>
+  func.return
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: power_op_test_si64
+func.func @power_op_test_si64() {
+  %lhs = stablehlo.constant dense<[-1, -1, -3, -3, 0]> : tensor<5xi64>
+  %rhs = stablehlo.constant dense<[1, 0, -3, 3, 2]> : tensor<5xi64>
+  %result = stablehlo.power %lhs, %rhs : tensor<5xi64>
+  check.eq %result, dense<[-1, 1, 0, -27, 0]> : tensor<5xi64>
+  func.return
+}
+
+// -----
+
+// CHECK-LABEL: Evaluated results of function: power_op_test_f64
+func.func @power_op_test_f64() {
+  %lhs = stablehlo.constant dense<[-2.0, -0.0, -36.0, 5.0, 3.0, 10000.0]> : tensor<6xf64>
+  %rhs = stablehlo.constant dense<[2.0, 2.0, 1.1, 2.0, -1.0, 10.0]> : tensor<6xf64>
+  %result = stablehlo.power %lhs, %rhs : tensor<6xf64>
+  check.almost_eq %result, dense<[4.000000e+00, 0.000000e+00, 0xFFF8000000000000,
```

Comment:
Is `0xFFF8000000000000` the value produced by the APFloat API or arbitrary?

Reply:
It is the actual result.